README.md

```diff
@@ -471,8 +471,7 @@ resty.kong.log.get\_log\_level
 
 Returns the current dynamic log level, remaining timeout and the original log level.
 
-If the dynamic log level is not set or not active,
-`timeout` will be `0`.
+`timeout` is in whole seconds, as an integer. If the dynamic log level is not set or not active, `timeout` will be `0`.
```

Comment:
``` suggestion
Returns the current dynamic log level, remaining timeout (in seconds), and the original log level.

If the dynamic log level is not set or not active,
the `timeout` will be `0`.
```

Reply:
<img width="766" alt="image" src="https://github.com/Kong/lua-kong-nginx-module/assets/623062/e6cc4f19-8d66-49e0-a170-480f8795a0fb">

hmm.

Reply:
fixed in new commit